llvm/include/llvm/IR/FPEnv.h

```diff
@@ -43,6 +43,15 @@ enum ExceptionBehavior : uint8_t {
 
 }
 
+inline bool isValidExceptionBehavior(unsigned X) {
```

Comment:
Do you know why the exception behaviors aren't handled in FloatingPointMode.h?

Reply:
I think this is historically, because the functionality was implemented by different people. We have "FPE.h" and "FloatingPointMode.h", both describing FP environment.